Tests/Repository/Vcs/GitDriverTest.php

```diff
@@ -102,6 +102,66 @@ public function testPublicRepositoryWithEmptyComposer($type, $filename)
      * @param string $type
      * @param string $filename
      */
+    public function testPublicRepositoryWithSkipUpdate($type, $filename)
+    {
+        $repoUrl = 'https://github.com/fxpio/composer-asset-plugin.git';
+        $identifier = '92bebbfdcde75ef2368317830e54b605bc938123';
+        $io = $this->getMockBuilder('Composer\IO\IOInterface')->getMock();
+
+        $this->config->merge([
+            'config' => [
+                'fxp-asset' => [
+                    'git-driver' => [
+                        'skip-update' => '1 week',
+                    ],
+                ],
+                'cache-vcs-dir' => sys_get_temp_dir().'/composer-test-cache-vcs',
+            ],
+        ]);
+
+        $repoConfig = array(
+            'url' => $repoUrl,
+            'asset-type' => $type,
+            'filename' => $filename,
```

Comment:
You must add the `asset-repository-manager` key in each repoConfig of failed tests.